internal/hardwaremanager/controller/helpers_test.go

```
@@ -815,10 +815,11 @@ var _ = Describe("Helpers", func() {
 	Describe("Validation Functions", func() {
 		var (
 			testBMH       *metal3v1alpha1.BareMetalHost
-			testHwProfile *hwmgmtv1alpha1.HardwareProfile
-			testHFC       *metal3v1alpha1.HostFirmwareComponents
-			testHFS       *metal3v1alpha1.HostFirmwareSettings
-			testClient    client.Client
+			testHwProfile  *hwmgmtv1alpha1.HardwareProfile
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)